File: app/src/main/java/de/robv/android/xposed/XposedHelpers.java
package de.robv.android.xposed;

import java.lang.reflect.Member;

/** Stub — replaced by LSPosed at runtime */
public class XposedHelpers {
    public static Member findAndHookMethod(Class<?> clazz, String methodName, Object... parameterTypesAndCallback) {
        return null;
    }
    public static Class<?> findClass(String className, ClassLoader classLoader) {
        try { return Class.forName(className, false, classLoader); }
        catch (ClassNotFoundException e) { return null; }
    }
}

File: app/src/main/java/de/robv/android/xposed/XposedHelpers.kt
package de.robv.android.xposed

import java.lang.reflect.Member

/** Stub — replaced by LSPosed at runtime */
object XposedHelpers {
    @JvmStatic
    fun findAndHookMethod(clazz: Class<*>, methodName: String, vararg parameterTypesAndCallback: Any): Member? = null

    @JvmStatic
    fun findClass(className: String, classLoader: ClassLoader): Class<*> =
        Class.forName(className, false, classLoader)
}
